src/Features/LanguageServer/Protocol/IRoslynLspLogger.cs

```diff
@@ -2,11 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
+using Microsoft.CommonLanguageServerProtocol.Framework;
 
 namespace Microsoft.CodeAnalysis.LanguageServer
 {
-    internal interface ILanguageServerTarget : IAsyncDisposable
+
+    internal interface IRoslynLspLogger : ILspLogger, ILspService
```

Comment:
This comment is similar to my other. The name Roslyn in the type doesn't add anything here. Would `ILspServiceLogger` maybe be more meaninful?